src/sonic-config-engine/translate_acl

```diff
@@ -1,174 +0,0 @@
-#!/usr/bin/env python
```

Comment:
@taoyl-ms , we still need to generate config db acl from the openconfig acl format, right?

Reply:
@lguohan ACL loader works with files in openconfig acl format.

Reply:
@lguohan acl-loader input file example:
https://github.com/Azure/sonic-mgmt/pull/299/files#diff-b72b79c3178d5157ef088fe15870f99e